src/Files.App/Views/SettingsPages/Tags.xaml

```diff
@@ -1,254 +1,321 @@
 ﻿<Page
-	x:Class="Files.App.SettingsPages.Tags"
```

Comment:
Can you set the spacing to tabs? You can change the default in your settings for Visual Studio.